test/electron/fixtures/app/index.html

```diff
@@ -59,5 +59,10 @@ <h2>actions</h2>
         <li><a href="#" id="main-process-clear-feature-flags-now" onclick="RunnerAPI.mainProcessClearFeatureFlagsNow()">immediately clear feature flags in main process</a></li>
         <li><a href="#" id="renderer-clear-feature-flags-now">immediately clear feature flags in renderer</a></li>
     </ul>
+    <ul>
+        <li><a href="#" id="renderer-clear-user">Clear user data set in renderer config</a></li>
+        <li><a href="#" id="renderer-clear-context">Clear context set in renderer config</a></li>
+        <li><a href="#" id="renderer-clear-metadata">Clear metadata set in renderer config</a></li>
```

Comment:
Actually, maybe this is clearer for the metadata link?

```suggestion
        <li><a href="#" id="renderer-clear-metadata">Clear "renderer" metadata section</a></li>
```

Reply:
Updated